modules/gapi/src/compiler/passes/pattern_matching.cpp

```diff
@@ -560,7 +569,7 @@ cv::gimpl::findMatches(const cv::gimpl::GModel::Graph& patternGraph,
         // Create vector with the correctly ordered OUT data nodes in the test subgraph
         std::vector<ade::NodeHandle> outputTestDataNodes;
         for (const auto& patternOutNode : patternOutputDataNodes) {
-            outputTestDataNodes.push_back(outputApiMatch[patternOutNode]);
+            outputTestDataNodes.push_back(outputApiMatch.at(patternOutNode));
```

Comment:
could you please consolidate these changes on the other code list when it is required?

Reply:
found only 1 different place, replaced there as well